templates/workspace_services/guacamole/user_resources/guacamole-azure-import-reviewvm/apply_patch.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Version of TRE to use as reference
+AZURE_TRE_VERSION="0.5.0"
```

Comment:
is this the right version now that we've release 0.5.1?

Reply:
I don't think it hugely matters but I can update this to 0.5.1

Reply:
I think it's worth doing. As you say, it shouldn't make any difference, but it prevents future questions.